src/icesat2_tracks/app.py

```diff
@@ -60,5 +64,15 @@ def makespectra(
     run_job(_makespectra, track_name, batch_key, ID_flag, output_dir)
 
 
+@app.command(help=_plotspectra.__doc__)
+def plotspectra(
```

Comment:
```suggestion
def plot_spectra(
```
Separate the words using an underscore.

Reply:
I could make this change. But in the interest of keeping name consistency, the corresponding [cli command](https://github.com/brown-ccv/icesat2-tracks/blob/84941b432ed3fd68e4364df656174126b22ca4eb/pyproject.toml#L162) is named `plotspectra` (no `_` or `-`) so users don't have to type as much. What do you think?

Reply:
I was checking the python naming conventions, but it seems the command line names have a different kind of nomenclature.
Also, It seems like typer transforms `_`  in a @app name's function  to a `-`  in the command line.  The  `delete_all`  in the example [here](https://typer.tiangolo.com/tutorial/commands/options/) . Would that work ?
@hollandjg  any thoughts ?

Reply:
I'd be tempted to make them all consistently `names_with_underscores` (and it sounds like those would become `names-with-underscores` on the command line) so that it's totally clear what is what – the name on the command line refers to a python function with a corresponding name.

Reply:
This patch will be applied with #111.